client/src/css/DiagnosticContainer.css

```diff
@@ -2,14 +2,15 @@
     .DiagnosticContainer {
         padding-top: 3vh;
         text-align: center;
-        height: 100vh;
+        overflow: scroll;
+        padding-bottom: 30vh;
     }
 
     .LogoRow {
         height: 15vh;
     }
-    
-    
+
+
```

Comment:
Try to avoid these types of changes so we can keep PRs small and contextualized

Reply:
Gotcha. I can change the PR but I'm not sure I can do the screenshot at the moment. I merged in the most up to date version of master and something in it is broken. I'll have to ask in discord about that.

Reply:
Updated!